.travis.yml

```diff
@@ -3,17 +3,28 @@ language: php
 
 php:
   - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
 
-env:
-  - SYMFONY_VERSION: 2.3.*
-  - SYMFONY_VERSION: 2.4.*
-  - SYMFONY_VERSION: 2.5.*
-  - SYMFONY_VERSION: 2.6.*
-  - SYMFONY_VERSION: 2.7.*@dev
+matrix:
+  allow_failures:
+    - php: 7.0
```

Comment:
This PR is mainly about PHP 7. It's been sent to other repos of this organization without this option. Feel free to send new PRs.